docs/adoptopenjdk.md

```diff
@@ -21,8 +21,8 @@ range of platforms. For information about the platforms and minimum operating sy
 
 If you obtain binaries from the AdoptOpenJDK community, the following pre-requisites are required:
 
-- OpenJDK binaries for Linux and AIX platforms from the AdoptOpenJDK community no longer bundle the OpenSSL cryptographic library. The library is expected to be found on the system path. If you want to use OpenSSL cryptographic acceleration, you must install OpenSSL 1.0.2 or 1.1.X on your system. If the library is not found on the system path, the in-built Java crytographic implementation is used instead, which performs less well.  
+- From Eclipse OpenJ9 release 0.13.0, OpenJDK binaries for Linux and AIX platforms from the AdoptOpenJDK community no longer bundle the OpenSSL cryptographic library. The library is expected to be found on the system path. If you want to use OpenSSL cryptographic acceleration, you must install OpenSSL 1.0.2 or 1.1.X on your system. If the library is not found on the system path, the in-built Java crytographic implementation is used instead, which performs less well.  
 - ![Start of content that applies only to Java 8](cr/java8.png) On Linux systems, the `fontconfig.x86_64` package should be installed to avoid a `NullPointerException` error when the AWT font subsystem is initialized. Work is ongoing at the AdoptOpenJDK project to fix this issue for their OpenJDK binaries.
-
+- From Eclipse OpenJ9 release 0.16.0 (OpenJDK 13) and release 0.17.0 (OpenJDK 8 and 11), CUDA is now enabled on Windows (x86-64) and Linux (x86-64 and IBM POWER LE) platforms, which allows you to offload certain Java application processing tasks to a general purpose graphics processing unit (GPU). To take advantage of this feature, your system must support NVIDIA Compute Unified Device Architecture (CUDA). The JIT requires the CUDA Toolkit 7.5 and your GPU device must have a minimum compute capability of 3.0. 
```

Comment:
CUDA is now enabled => GPU support is now enabled?

Reply:
>CUDA is now enabled => GPU support is now enabled?

Those are @keithc-ca 's words @gita-omr and he is typically very precise. Keith?

Reply:
The are two components that take advantage of CUDA GPUs: the CUDA4J API and the JIT. Gita's focus is on the JIT where it has always been discussed as 'GPU support', so I can see where her suggestion comes from. However, this section relates to both components, so I think CUDA is the better term - it's more specific than 'GPU' and appropriate because only NVIDIDA GPUs are supported - those with CUDA capabilities.